sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/dsl/package.scala

```diff
@@ -528,6 +528,8 @@ package object dsl {
       }
 
       def deduplicate(colNames: Attribute*): LogicalPlan = Deduplicate(colNames, logicalPlan)
+
+      def distinct(): LogicalPlan = Deduplicate(logicalPlan.output, logicalPlan)
```

Comment:
seems this is not used?

Reply:
oh yes because I had to use DataFrame API for testing this ime.

Removed.